File: RAS/server/service/src/main/java/ua/softserve/service/FeedbackService.java
package ua.softserve.service;


import ua.softserve.persistence.entity.Feedback;

import java.util.List;

public interface FeedbackService {
    /**
     * Update feedback of Student.
     */
    void updateFeedback(Feedback feedback);

    /**
     * Update interviewer comment of Student.
     */
    void updateInterviewerComment(String comment, Integer studentId);
}

File: RAS/client/src/app/services/feedbacks/feedback.service.ts
import {Injectable} from '@angular/core';
import {HttpClient} from '@angular/common/http';
import {Observable} from 'rxjs/Observable';
import {environment} from "../../../environments/environment";
import {Feedback} from "../../models/feedbacks/student.model";


@Injectable()
export class FeedbackService {

  constructor(private http: HttpClient) {}

  updateFeedback(feedback: Feedback): Observable<any> {
    let url = environment.serverUrl+'feedback/update';
    return this.http.put(url,feedback, {withCredentials: true});
  }

  updateInterviewerComment(interviewerComment: string, studentId: number): Observable<any> {
    let url = environment.serverUrl + 'feedback/update_comment/'+ studentId;
    return this.http.put(url, interviewerComment,  {withCredentials: true});
  }
}
